sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -117,7 +117,6 @@ public final class ConfigurationClientBuilder implements
     ConfigurationTrait<ConfigurationClientBuilder>,
     EndpointTrait<ConfigurationClientBuilder> {
     private static final RetryPolicy DEFAULT_RETRY_POLICY = new RetryPolicy("retry-after-ms", ChronoUnit.MILLIS);
-    private static final SyncTokenPolicy DEFAULT_SYNC_TOKEN_POLICY = new SyncTokenPolicy();
 
```

Comment:
It's not letting me comment on the lines but could perCallPolicies and perRetryPolicies being shared ever cause and issue?

Reply:
`perCallPolicies`  and `perRetryPolicies` are shared and can be updated per client builder.  There is a method
```
public ConfigurationClientBuilder addPolicy(HttpPipelinePolicy policy)
```
So we need to keep it as shared per builder.